.travis.yml

```diff
@@ -1,5 +1,10 @@
 language: node_js
 node_js:
     - "6"
+addons:
+  sauce_connect: true
+before_script:
```

Comment:
This runs a simple python server before we start running our job so that we can point sauce labs to a "hosted" version of preview.